src/libs/bridge/ReactNativeLauncherMessenger.js

```diff
@@ -15,7 +15,7 @@ export default class ReactNativeLauncherMessenger extends MessengerInterface {
   }
```

Comment:
can you add the `why` in the commit message? What kind of messages are logged that we don't want anymore?